frontend/src/views/SQR/Form.vue

```diff
@@ -266,7 +266,7 @@ export default class Form extends Vue {
     } else if (
       this.$ability.cannot(
         "submit",
-        new SQRForm({
+        new SQRFormAbilityData({
           isCompleted: allotmentStatus.toLowerCase() === "done"
```

Comment:
Why can't we use `this.isCompleted` here as well? Actually, checking the allotment status is correct, then what is `this.isCompleted`?

Reply:
`this.isCompleted` returns true when data is loaded from branch `completed` or `migrated`.

Reply:
I feel that the proper way is to check the allotment status rather than the source branch of the form. Is it possible for both places?

And it might be better to rename the `isCompleted` property to something like `isLoadedFromCompletedBranch` for clarity.

Reply:
Moreover! I have just realized that it is absolutely improper to rely on the source branch, as the form may be completed (submitted) and still it is available for re-submission by the author until it is marked as Done by the checker. So we should rely on the allotment status exclusively.